### PR TITLE
[OTTO-2667] Add CLI error message

### PR DESCRIPTION
**Pull request by joa-pan**
Closes #

## Summary

**[Doc Page Title](https://pantheon.io/docs/doc-title)** - <Enter a one sentence summation of the pertinent changes (including not-yet-completed work) provided by this PR.>

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

File: source/content/guides/autopilot/05-troubleshoot.md [hunk 1]
```diff
@@ -343,3 +343,47 @@ Site is running a framework that is not Drupal, Drupal 8, or WordPress. Currentl
...
+
+<Accordion title="We ran into an issue with Autopilot because the database could not be updated." id="no-database-updates" icon="info-sign">
+
+### Issue
+
+An issue with Autopilot occurred because the WordPress Dev, Test, or Live database could not be updated. 
+  
+### Solution
+  
+Try updating the database locally, and then retry Autopilot. Contact Support[/link] for assistance if the issue persists.
```
[the remark this answers is not in the document]
Comment (pwtyler) [reply source/content/guides/autopilot/05-troubleshoot.md:387]: Yes, perhaps, "on the target environment"?

This kind of failure is fairly rare IMO, but it's certainly possible, hence the event.

File: source/content/guides/autopilot/05-troubleshoot.md [hunk 2]
```diff
@@ -343,3 +343,47 @@ Site is running a framework that is not Drupal, Drupal 8, or WordPress. Currentl
...
+
+## CLI Tool Failed
+
+<Accordion title="We ran into an issue with Autopilot because Drush or WP-CLI did not work as expected." id="failed-cli-tool" icon="info-sign">
+
+### Diagnosis
+
+There is an issue with Autopilot because Drush or WP-CLI did not work as expected in the Dev or Autopilot environment. 
+
+Run `terminus remote:drush $SITE.dev -- pml` or `terminus remote:wp $SITE.dev -- plugin` to list and check for unexpected output or errors. Alternatively you can use the command `$SITE.autopilot` for the Autopilot environment
```
Comment (pwtyler) [inline source/content/guides/autopilot/05-troubleshoot.md:355]:
```suggestion
Run `terminus remote:drush $SITE.dev -- pml` or `terminus remote:wp $SITE.dev -- plugin list` and check for unexpected output or errors. Alternatively you can use the command `$SITE.autopilot` for the Autopilot environment
```

File: source/content/guides/autopilot/05-troubleshoot.md [hunk 3]
```diff
@@ -343,3 +343,47 @@ Site is running a framework that is not Drupal, Drupal 8, or WordPress. Currentl
...
+  
+### Solution
+  
+After diagnosing the problem, resolve the issue that is causing the fatal error on the Dev environment, then retry Autopilot. For additional help contact Support[/support] for assistance.
```
Comment (pwtyler) [inline source/content/guides/autopilot/05-troubleshoot.md:359]:
```suggestion
After diagnosing the problem, resolve the issue that is causing the error on the Dev or Autopilot environment, then retry Autopilot. For additional help contact Support[/support] for assistance.
```

File: source/content/guides/autopilot/05-troubleshoot.md [hunk 4]
```diff
@@ -343,3 +343,47 @@ Site is running a framework that is not Drupal, Drupal 8, or WordPress. Currentl
...
+  
+</Accordion>
+
+## Quicksilver is Unavailable
+
+<Accordion title="We could not execute some Quicksilver scripts on this site." id="no-quicksilver" icon="info-sign">
+
+### Issue
+
+Quicksilver scripts could not be execute on the site.
```
Comment (pwtyler) [inline source/content/guides/autopilot/05-troubleshoot.md:369]:
```suggestion
Quicksilver scripts failed to execute on the site. 
```

File: source/content/guides/autopilot/05-troubleshoot.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (pwtyler) [reply source/content/guides/autopilot/05-troubleshoot.md:387]: The customer can use `wp core update-db` from the CLI, or will be prompted to run the update on the admin dashboard of the site.

**Review by pwtyler — APPROVED** [review]

**Comment by joa-pan** [comment]
lgtm

**Pull request merged**